Update the header of every file - new comments (#108188)
AdguardTeam/AdguardFilters: TrackParamFilter/sections/whitelist.txt

!

> ! This section contains the list of the rules that fix incorrect blocking. Rules should be specific and with only one modifier `removeparam`.

Reviewer: ```suggestion
! This section contains the list of rules that fix incorrect blocking. Rules should be domain-specific and only the `removeparam` modifier is allowed.
```
